scripts/auto-rebase/rebase_gateway_api.sh

```diff
@@ -91,20 +91,12 @@ rebase_ossm_to() {
 
     update_last_ossm_rebase "${operator_bundle_manifest}" "${gateway_api_version}"
   
-    update_ossm_images
-    if [[ -n "$(git status -s pkg/release)" ]]; then
-        title "## Committing changes to pkg/release"
-        git add pkg/release
-        git commit -m "update OSSM images"
-    else
-        echo "No changes in OSSM images."
-    fi
-
     update_ossm_manifests "${operator_bundle_manifest}"
+    update_ossm_images
```

Comment:
What about `release-gateway-api-$arch.json` file for OSSM?

Reply:
This one is going to be part of a different PR, I wanted to establish this one, then tests, then the release-info. Those files need to be created manually and updated automatically (they are not included in the bundle, but there is data there to feed them)